### PR TITLE
Check token scope install script

### PR DESCRIPTION
**Pull request by DTMad**

### CODE REVIEW

File: deploy/install.sh [hunk 1]
```diff
@@ -176,18 +175,59 @@ EOF
...
+    exit 1
+  fi
+}
+
+apiRequest() {
+  method=$1
+  url=$2
+  json=$3
+
+  response="$(curl -sS -X ${method} "${API_URL}""${url}" \
```
Comment (lrgar) [inline deploy/install.sh:219]:
```suggestion
  response="$(curl -sS -X ${method} "${API_URL}${url}" \
```

File: deploy/install.sh [hunk 2]
```diff
@@ -176,18 +175,59 @@ EOF
...
+  fi
+
+  if ! echo "$responseAPI" | grep -q "ReadConfig"; then
+    echo "Error: API token does not have config read permission!"
+    exit 1
+  fi
+
+  responsePaaS=$(apiRequest "POST" "/v1/tokens/lookup" "${jsonPaaS}")
+
+  if echo "$responsePaaS" | grep -q "Token does not exist"; then
```
Comment (lrgar) [inline deploy/install.sh:208]: Technically, you could check for the `InstallerDownload` permission for PaaS tokens.

**Review by lrgar — DISMISSED** [review]

File: deploy/install.sh (hunk 2 above)
Comment (DTMad) [reply deploy/install.sh:208]: Since the PaaSToken can only have InstallerDownload permission checking for it's existence should be sufficient here

Comment (lrgar) [reply deploy/install.sh:208]: Except when the customer uses an API token as PaaS token, which is not that uncommon.

**Review by lrgar — APPROVED** [review]

**Pull request merged**